tools/testbuild.sh

```diff
@@ -258,6 +258,10 @@ function build {
     fail=1
   fi
 
+  # Verify that the context rule does not need to be rebuilt
+
+  makefunc context --question
```

Comment:
should we generate some log in the error case?

Reply:
What will be our plan here?  Should we add other targets as well?  Or create separate workflow for build system testing?

Reply:
I think that @AlanRosenthal just want to check "make context" doesn't do any action after the initial config, but it's hard to identify this issue if there is no any output to indicate the problem.

Reply:
I understand the purpose, but what I am saying is, are we going to generalize this kind of tests for the build system? It seems a bit out of scope of `testbuild.sh`.

Reply:
Why out of scope? 'make context' is also part of build process, I think.

Reply:
It is part of the build system, but in `testbuild.sh` we don't unit test the build system, it just builds a bunch of configurations.
I don't have a strong feeling about this, but I wanted to know if we would want to add other parts of the build system here.